### DISCUSSION TITLE
Github Issues for linuxbrew-core - is the "New issue on Homebrew/linuxbrew-core" just an infinite loop?

### DISCUSSION THREAD

**Discussion by mellery451** [Homebrew/discussions#858] [Linux] [answered]
regarding [github issues for linuxbrew](https://github.com/Homebrew/linuxbrew-core/issues/new/choose) - does the last option work for anybody?

![Screen Shot 2021-02-18 at 1 50 41 PM](https://user-images.githubusercontent.com/463430/108426375-873ddc00-71f0-11eb-9ac2-bde84f42eaa4.png)

When I select it, just takes me to the same page? Am I missing something

**Comment by Rylan12** [comment, marked as answer]
This is just a byproduct of our merge process between Homebrew/homebrew-core and Homebrew/linuxbrew-core.

I've opened a PR: https://github.com/Homebrew/linuxbrew-core/pull/22316

**Reply by Rylan12** [reply]
You should now be able to file a bug report in the Homebrew/linuxbrew-core repo.

**Reply by mellery451** [reply]
looks good - thanks
